Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "asset-classification-smart-contract"
-version = "0.0.12"
+version = "0.0.13"
```

Comment:
I totally remembered to do this this time.  Also, unless we run into problems with rolling this thing out, I feel like we're dang close to v1.0.0